pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/DemoPlayer.kt

```diff
@@ -21,17 +20,17 @@ import ch.srgssr.pillarbox.ui.ScaleMode
  *
  * @param player The player to display.
  * @param modifier The modifier to be applied to the layout.
- * @param enableUi Enable User interface and subtitles
  * @param scaleMode The scale mode to use.
  * @param defaultAspectRatio The aspect ratio to use while video is loading or for audio content.
+ * @param surfaceContent The Composable surface content.
  */
 @Composable
 fun DemoDefaultPlayer(
     player: Player,
     modifier: Modifier = Modifier,
-    enableUi: Boolean = false,
     scaleMode: ScaleMode = ScaleMode.Fit,
-    defaultAspectRatio: Float? = null
+    defaultAspectRatio: Float? = null,
+    surfaceContent: @Composable (() -> Unit)? = null
```

Comment:
⚠️ **[Android Lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>ComposableLambdaParameterNaming: Composable lambda parameter should be named `content`